spec/features/cycle_counts/user_creates_a_new_cycle_count_spec.rb

```diff
@@ -18,7 +18,7 @@
 
       expect(page).to have_content(I18n.t('cycle_counts.create.success'))
       expect(page).to have_content(locations[0].description)
-      expect(page).to have_content(requested_date)
+      expect(page).to have_content(requested_date.strftime('%m/%d/%Y'))
```

Comment:
I don't understand.  The object isn't brought in to decorate.  I'm trying to make sure that given(:requested_date) { Faker::Date.forward(30) } gets changed to what the formatted_requested_date would be.  Unless I create another given like this? given(:formatted_requested_date) { requested_date.strftime('%m/%d/%Y') }